### PR TITLE
Care home open data

### PR DESCRIPTION
**Pull request by Moohan**
I was working on this code for the June update, to start using the LA-level costs... That part of it fell through but I'd also made some changes to use the Open Data Platform.

The main changes are getting the correct data from the open data platform and filling in 2019/20 data (we did this in the CH_costs.xlsx spreadsheet previously).

### CODE REVIEW

File: Production_scripts/Create_costs/costs-care_home.R [hunk 1]
```diff
@@ -29,40 +30,48 @@ fs::file_copy(get_ch_costs_path(),
...
 
-## Read excel data
-ch_costs_data <- readxl::read_xlsx(
-  paste0(get_slf_dir(), "/Costs/CH_Costs.xlsx")
-)
+## Read costs from the CHC Open data
+ch_costs_data <- phsopendata::get_resource(
+  res_id = "4ee7dc84-ca65-455c-9e76-b614091f389f",
+  col_select = c("Date", "KeyStatistic", "CA", "Value")
+) %>%
```
Comment (catherineholland1) [inline Production_scripts/Create_costs/costs-care_home.R:37]: this did not work for me, error with `col_select` is there a missing package? 

alternative that works is:
```suggestion
ch_costs_data <- phsopendata::get_resource(
  res_id = "4ee7dc84-ca65-455c-9e76-b614091f389f"
) %>%
select(Date, KeyStatistic, CA, Value) %>%
```

File: Production_scripts/Create_costs/costs-care_home.R [hunk 2]
```diff
@@ -88,9 +97,7 @@ ch_costs_uplifted <-
 # Join data together  -----------------------------------------------------
 
 # match files - to make sure costs haven't changed radically
-old_costs <- haven::read_sav(
-  get_ch_costs_path(update = latest_update())
-) %>%
+old_costs <- haven::read_sav(get_ch_costs_path(update = latest_update())) %>%
```
Comment (catherineholland1) [inline Production_scripts/Create_costs/costs-care_home.R:100]: getting an error reading this file in

`Error: Failed to parse /tmp/Rtmp5rZFxS/file93037b8b0e45: Invalid file, or file has unsupported features.` 

can't see any change between the old way and new? has `get_ch_costs_path` changed?

**Review by catherineholland1 — CHANGES_REQUESTED** [review]
just a few wee things to check that are throwing up errors for me?

File: Production_scripts/Create_costs/costs-care_home.R (hunk 1 above)
Comment (Moohan) [reply Production_scripts/Create_costs/costs-care_home.R:37]: `col_select` is  in the latest version of `phsopendata` if you get the new version it should work 😄

File: Production_scripts/Create_costs/costs-care_home.R [hunk 3]
```diff
@@ -29,40 +30,48 @@ fs::file_copy(get_ch_costs_path(),
...
 )
 
-## Read excel data
-ch_costs_data <- readxl::read_xlsx(
-  paste0(get_slf_dir(), "/Costs/CH_Costs.xlsx")
-)
+## Read costs from the CHC Open data
+ch_costs_data <- phsopendata::get_resource(
+  res_id = "4ee7dc84-ca65-455c-9e76-b614091f389f",
+  col_select = c("Date", "KeyStatistic", "CA", "Value")
```
Comment (catherineholland1) [inline Production_scripts/Create_costs/costs-care_home.R:36]:
```suggestion
  phsopendata::col_select = c("Date", "KeyStatistic", "CA", "Value")
```

**Review by catherineholland1 — APPROVED** [review]
just add the `phsopendata` in front of the `col_select` to allow that to work then all good to merge

File: Production_scripts/Create_costs/costs-care_home.R (hunk 2 above)
Comment (catherineholland1) [reply Production_scripts/Create_costs/costs-care_home.R:100]: `gte_ch_costs_path()` still `.sav` on main-R so still needs merged

**Pull request merged**